config.json.example

```diff
@@ -50,6 +50,22 @@
         "sell_profit_only": false,
         "ignore_roi_if_buy_signal": false
     },
+    "edge": {
```

Comment:
Please also add new settings to the `config_full.json.example` file, and in addition to that, to the `CONF_SCHEMA` in `constants.py` so they are properly validated for correctness